go/cmd/topo2topo/plugin_kubernetestopo.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+// This plugin imports kubernetestopo to register the kubernetes implementation of TopoServer.
+
+import (
+	_ "vitess.io/vitess/go/vt/topo/kubernetestopo"
```

Comment:
The package name could be shortened to k8stopo (or k3stopo)?

Reply:
Agreed. All the flags and other config use `k8s` instead of `kubernetes`. I'll change it.